controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -332,6 +332,9 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (re
 		&ocsCephObjectStoreUsers{},
 		&ocsCephObjectStores{},
 		&ocsCephRbdMirrors{},
+		&ocsCephNFS{},
+		&ocsCephNFSService{},
+		&ocsCephNFSBlockPool{},
```

Comment:
Why does this implementation treat the NFS block pool separately from the other `CephBlockPools` yet treat the NFS StorageClass together with the other StorageClasses? Shouldn't all resources follow similar patterns?

@jarrpa do you have implementation preferences on this matter (ocs-op should create a CephBlockPool to configure the shared config location for all CephNFSes)?

Reply:
> Why does this implementation treat the NFS block pool separately from the other `CephBlockPools` yet treat the NFS StorageClass together with the other StorageClasses? Shouldn't all resources follow similar patterns?
> 
> @jarrpa do you have implementation preferences on this matter (ocs-op should create a CephBlockPool to configure the shared config location for all CephNFSes)?

moved it with cephblockpool